team.html

```diff
@@ -47,17 +47,35 @@
             <div class="collapse navbar-collapse" id="navbarSupportedContent">
                 <ul class="navbar-nav ml-auto">
                     <li class="nav-item active">
-                        <a class="nav-link font-weight-bold nav-items" href="index.html">Home <span
-                                class="sr-only">(current)</span></a>
+                        <a class="nav-link font-weight-bold nav-items" href="index.html">Home </a>
                     </li>
-
                     <li class="nav-item">
-                        <a class="nav-link  font-weight-bold nav-items" href="#team">Team</a>
+                        <a class="nav-link font-weight-bold nav-items" href="index.html #about">About</a>
```

Comment:
It should be `index.html#about` and not `index.html #about`

Reply:
ok I will do that